boards/jiminy-mega256rfr2/Makefile.include

```diff
@@ -0,0 +1,57 @@
+export CPU = atmega256rfr2
+export CPU_MODEL = atmega256rfr2
+
+export TERMPROG = $(RIOTBASE)/dist/tools/pyterm/pyterm
+export TERMFLAGS = -b 9600 -p $(PORT)
+
+#define the flash-tool and default port depending on the host operating system
```

Comment:
Not 100% sure, but do you think you could reuse the common arduino-atmega-common Makefile.include to configure the flasher ? Look at how it's done for arduino-uno.